.github/workflows/image_build_push.yml

```diff
@@ -0,0 +1,62 @@
+# This is a basic workflow to help you get started with Actions
+
+name: docker-image-push-join
+
+# Controls when the action will run. Triggers the workflow on push or pull request 
+# events but only for the master branch
+on:
+  push:
+    # Mukul:
+    # I've added a local test branch on my system and using it for testing image push.
+    # So, for testing purposes, need to checkout a branch "image-push-merge"
+    # TODO: Need to change to build off master or main once it looks good.
+    branches: [ image-push-merge, image-push ]
+
+
+# Env variable
+env:
+  DOCKER_USER: ${{secrets.DOCKER_USER}}
+  DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    # - name: docker login
+    #   run: | # log into docker hub account
+    #     docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+
+    - name: Get current date # get the date of the build
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d--%M-%S')"
+
+    - uses: actions/download-artifact@v4
+      with:
+        name: server-image-tag
+        github-token: ${{secrets.GH_PAT_IMAGE_PUSH}}
+        repository: MukuFlash03/e-mission-server
+        run-id: 8562709783
+
+    - name: Get artifact
+      run: ls -R 
+
+    # #Runs a single command using the runners shell
+    # - name: Run a one-line script
+    #   run: echo running in repo ${GITHUB_REPOSITORY#*/} branch ${GITHUB_REF##*/} on ${{ steps.date.outputs.date }}
+
+    # # Runs a set of commands using the runners shell
+    # - name: build docker image
+    #   run: |
+    #     docker build -t $DOCKER_USER/${GITHUB_REPOSITORY#*/}:${GITHUB_REF##*/}_${{ steps.date.outputs.date }} ./frontend
+    #     docker images
```

Comment:
if this is commented out, how were you able to test building images in GitHub actions?

Per @nataliejschultz 
> outside of running GitHub Actions, which we have proven to work